sstables/mp_row_consumer.hh

```diff
@@ -82,6 +82,10 @@ public:
     // to the next partition.
     virtual void on_out_of_clustering_range() = 0;
 
+    // Called when a clustering range is exhausted, but more clustering ranges
+    // remain (so a promoted index seek is needed).
+    virtual void on_clustering_range_change() = 0;
```

Comment:
It seems that you never call it from the ka/la parser.

Reply:
An omission like this would manifest in lower performance, since it's always ok to not call advance_context(), we have filtering in place.

May be worthwhile to compare perf_fast_forward diff for blatant regressions.

Reply:
I meant to, somehow I missed it. I'll check with p_f_f.